API Documentation.md

```diff
@@ -162,9 +162,51 @@ None
 | `time`           | datetime | A time string in ISO8601 format (Ex: 2020-08-11T19:29:25Z)   |
 | `tag_type`       | char     | A character denoting the type of tag, stun or kill. Should be 'S' or 'K', respectively. |
 
+#### Failure Response
+
+`401` - The user is not logged in.
+
+`403` - The user is not a player.
+
+## Get Player List
+
+`/api/v1/get_player_list/`
+
+Gets all players playing in the same game as the logged in user.
+
+#### Method
+
+`GET`
+
+#### Parameters
+
+None
+
+#### Body Parameters
+
+None
+
+#### Success Response
+
+`200` - Returns a JSON list of player information. Each `Player` in the list has the fields:
+
+##### Player
+
+| Name             | Type    | Description                                                  |
+| ---------------- | ------- | ------------------------------------------------------------ |
+| `roleChar`       | Char    | One character that represents the role of the player. `H` for human, `Z` for zombie, and `S` for spectators |
+| `name`           | String  | Player's name                                                |
+| `processedScore` | Integer | If the player has chosen to share their score publicly, this will be the player's score. Otherwise, a `-1` value here means that the score is hidden. |
```

Comment:
Hm I agree with this API showing a -1 if the score is hidden, but in the app I think we should show a 0 for the score. The reason for this is that it might give extra information we don't want the players to have, if that makes sense.

Reply:
I mean, in the app, we can just write "Hidden". The purpose of returning -1 is that anyone who uses the API gets the relevant information. How we display that information is up to whoever is accessing the API. At the moment, we only have one type of client (the app), but if someone else decides to build something for this, it would be up to them.